bin/BuildPackages.sh

```diff
@@ -236,17 +209,13 @@ build_one_package() {
       build_carat
     ;;
 
-    cohomolo*)
```

Comment:
I am not really using it myself -- I just run `BuildPackages.sh` and looked into all the errors it produced :-).